validation-test/Reflection/reflect_Optional_Bool.swift

```diff
@@ -0,0 +1,35 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -g -lswiftSwiftReflectionTest %s -o %t/reflect_Optional_Bool
+// RUN: %target-codesign %t/reflect_Optional_Bool
+
+// RUN: %target-run %target-swift-reflection-test %t/reflect_Optional_Bool | %FileCheck %s --check-prefix=CHECK-%target-ptrsize
```

Comment:
Nothing in this output should depend on the pointer size, so you should delete this `--check-prefix` flag, and change all the check comments to just `// CHECK:`. That should let it work seamlessly on 32-bit as well.